internal/checker/checker.go

```diff
@@ -15198,16 +15198,26 @@ func (c *Checker) getTypeOfVariableOrParameterOrPropertyWorker(symbol *ast.Symbo
 	if symbol == c.requireSymbol {
 		return c.anyType
 	}
-	// !!! Handle SymbolFlagsModuleExports
+	if symbol.Flags&ast.SymbolFlagsModuleExports != 0 && symbol.ValueDeclaration != nil {
```

Comment:
We're not currently creating any `SymbolFlagsModuleExports` symbols, but presumably we will as we implement JS/JSDoc checking. Otherwise, we can delete this if statement.